Algorithm.Python/QuantConnect.Algorithm.Python.csproj

```diff
@@ -54,6 +54,7 @@
     <Content Include="Alphas\TripleLeverageETFPairVolatilityDecayAlpha.py" />
     <Content Include="Alphas\VIXDualThrustAlpha.py" />
     <Content Include="BasicSetAccountCurrencyAlgorithm.py" />
+    <None Include="RawPricesUniverseRegressionAlgorithm.py" />
```

Comment:
Should this be `Content`?

Reply:
It doesn't need to be. We have both `None` and `Content` for these files.